shortfin/python/shortfin_apps/sd/components/service.py

```diff
@@ -514,17 +525,17 @@ async def _denoise(self, device, requests):
                 "".join([f"\n  {i}: {ary.shape}" for i, ary in enumerate(unet_inputs)]),
             )
             await device
-            (noise_pred,) = await fns["unet"](*unet_inputs)
+            (noise_pred,) = await fns["unet"](*unet_inputs, fiber=self.fiber)
             await device
```

Comment:
We should be able to take all of these explicit device waits out after my patch (#350) lands. We could use some more framework tests on dependencies to be sure but should be ok (all of my examples to you guys had waits in them so as to not trip you up while things were getting worked out).

Leave in for now and we can come back.